### ISSUE TITLE
Remove PS Team's migrated content from DevHub

### ISSUE THREAD

**Issue by mitovskaol** [BCDevOps/developer-experience#2757]
Now that the new Tech Docs site (https://beta-docs.developer.gov.bc.ca) is live, we need to remove any of our duplicate content from DevHub (https://developer.gov.bc.ca) and direct visitors to the new site.

Use [this spreadsheet](https://docs.google.com/spreadsheets/d/1EVFGwxQgdHZPtS2KzdKbJlJqQVDQzomKIe1Y62VpqcI/edit#gid=804654598) to check if the content is owned by an external owner. **External content should remain in DevHub index files until the current DevHub is replaced with a new product.**

Scope: this ticket applies to the content owned by the Platform Services team only and does not include the removal of the externally owned content. 

## Additional context
Google Sheet of externally owned content in DevHub: https://docs.google.com/spreadsheets/d/1EVFGwxQgdHZPtS2KzdKbJlJqQVDQzomKIe1Y62VpqcI/edit#gid=804654598

DevHub repository: https://github.com/bcgov/devhub-app-web

This issue is concerned with the Gatsby project in the `web` folder of the repository. This app generates the static site that users see when they visit DevHub. The `web` folder includes a detailed [README with instructions for local development](https://github.com/bcgov/devhub-app-web/tree/master/web#deployment-local-development-for-app-web). When creating your `.env.production` file from `.env.production.example`, please see @ty2k for environment variables.

When we refer to "removing migrated content", we want to remove content from the deployed DevHub site **while adding placeholder text at the existing URLs**. This is important - our users have documentation pages bookmarked, and they are indexed by search engines. If we were to remove a page entirely without providing placeholder content, links to that page will 404. We want to keep the same URL but change the underlying content to a placeholder indicating that the new content lives at Tech Docs.

DevHub hosts documents generated from its own repository, but also from external repositories owned by the Platform Services team. 

If you are replacing an internal document within the project, you can remove the old content while updating the `topicRegistry` to point to a new placeholder file.

 If you are replacing a document from an external repository, you should remove the reference to it from a given `topicRegistry` file, and also create a new internal document **that will use the same URL**. 

URLs are case-sensitive and generated based on the title of the Markdown document (as determined by the first `#` / `<h1>` heading). Because we are generating URLs based on the document titles, it's necessary to create/replace documents for every page we are retiring. In other words, it's not enough to go into each `topicRegistry` file and update the file reference to point to a single placeholder page - we need one placeholder page for each document being retired, ensuring that the titles for the new placeholder page are identical to the page being replaced.

One quirk of DevHub URLs is that content pages tend to live at both `/<page title>` and also `/<topic>/<page title>`. Neither of these are declared canonical, and both use cases will have potentially been bookmarked by users or indexed by search engines. The `/<page title>` format of a page gets surfaced through search, while the `/<topic>/<page title>` format is navigated to through the "topic" landing page UI.

It would be best to create the new placeholder documents inside of the DevHub repo, as opposed to creating new dependencies on external repos.

**Example external page**: "12 Factor Apps"
- Content is in an external repository: https://github.com/bcgov/devhub-resources/blob/master/resources/community/12-factor-apps.md
- The Markdown file is referenced in a `topicRegistry` file: https://github.com/bcgov/devhub-app-web/blob/master/web/topicRegistry/getting-started-on-the-devops-platform.json#L23
- The rendered page is served from two different URLs:
  - https://developer.gov.bc.ca/12-Factor-Apps
  - https://developer.gov.bc.ca/Getting-Started-on-the-DevOps-Platform/12-Factor-Apps

**Example internal page**: "Integrating Algolia Search Engine Into the Devhub"
- Content is in the DevHub repository: https://github.com/bcgov/devhub-app-web/blob/master/web/src/assets/blog/integratingAlgolia.md
- The Markdown file is referenced in a `topicRegistry` file: https://github.com/bcgov/devhub-app-web/blob/master/web/topicRegistry/community-contributed.json#L20
- The rendered page is served from two different URLs:
  - https://developer.gov.bc.ca/Integrating-Algolia-Search-Engine-Into-the-Devhub
  - https://developer.gov.bc.ca/Community-Contributed-Content/Integrating-Algolia-Search-Engine-Into-the-Devhub
 
## Definition of done
- [x] Update DevHub index files to remove references to the content owned by Platform Services Team repo.   
- [x] Add a placeholder for the removed pages that indicates that the content has moved. The placeholder can be:
```markdown
The content you are trying to access has moved. If it is technical content, look for it on [BC Gov Private Cloud Platform's Technical Documentation website](https://beta-docs.developer.gov.bc.ca/). For non-technical content, check out our [new BC Gov Private Cloud Platform website](https://cloud.gov.bc.ca/private-cloud).

If you can't find the information you are looking for on either of these sites, please email the Platform Services team (<a href="mailto:PlatformServicesTeam@gov.bc.ca">PlatformServicesTeam@gov.bc.ca</a>).
```

**Comment by ty2k** [comment]
This ticket is currently blocked by broken builds in [bcgov/devhub-app-web PR #1766](https://github.com/bcgov/devhub-app-web/pull/1766).

**Comment by ty2k** [comment]
With builds now succeeding, this ticket is no longer blocked, and we are able to change our `topicRegistry` JSON configuration files to reference our newly merged placeholder documents from [bcgov/devhub-app-web PR #1766](https://github.com/bcgov/devhub-app-web/pull/1766).

**Comment by ty2k** [comment]
The pull request [bcgov/devhub-app-web #1766](https://github.com/bcgov/devhub-app-web/pull/1771) is where the individual page references are being switched to point to the new boilerplate content. This pull request was deployed to `dev` and I gave our internal chat channel notice on Friday, September 9 that this content migration effort was going live. Today, I deployed to `test` and `prod` with the intention of merging this PR as soon as possible, in order to start a new PR where I can deploy the "If you can't find..." text from the master branch.

**Comment by ty2k** [comment]
In pull request [bcgov/devhub-app-web #1773](https://github.com/bcgov/devhub-app-web/pull/1773), I've deployed all of @Kolezhanchik's changes (including the full boilerplate text) to production, and I believe this can be closed.
